component/client-handler/src/main/java/org/wso2/carbon/identity/oauth2/token/handler/clientauth/jwt/validator/JWTValidator.java

```diff
@@ -91,7 +91,7 @@ public class JWTValidator {
 
     private JWTStorageManager jwtStorageManager;
 
-    public JWTValidator(boolean preventTokenReuse, String validAudience, int rejectBefore, String validIssuer,
+    public JWTValidator(boolean preventTokenReuse, List<String> validAudience, int rejectBefore, String validIssuer,
```

Comment:
We can't change a public method signature. We need to keep the old definition as it is and define as @Deprecated and call the new method.

Reply:
Ack. Readded the old method and deprecated it.